core/plugins/NtupleWriter.cc

```diff
@@ -1315,21 +1341,33 @@ bool NtupleWriter::filter(edm::Event& iEvent, const edm::EventSetup& iSetup) {
 
   if(doL1seed && iEvent.isRealData()){
     auto & triggerResults = *event->get_triggerResults();   
+    auto & triggerPrescales = *event->get_triggerPrescales();
+    auto & triggerPrescalesL1min = *event->get_triggerPrescalesL1min();
+    auto & triggerPrescalesL1max = *event->get_triggerPrescalesL1max();
+
     //muGT 
     edm::Handle<BXVector<GlobalAlgBlk>> l1GtHandle;
     iEvent.getByToken(l1GtToken_, l1GtHandle);
     bool prefire = l1GtHandle->begin(-1)->getFinalOR();
     triggerNames_outbranch.push_back("muGT_BX_minus_1__prefire");
     triggerResults.push_back(prefire);
+    triggerPrescales.push_back(1);
```

Comment:
Is there a reason for storing 1s for all these prescales?

Reply:
Yes, in the code it assumes that triggerResults and triggerPrescales size is the same. Dummy values are stored to avoid changes in the code  and keep it compatible with old ntuples.